src/ResultWriter/DefaultResultWriter.php

```diff
@@ -13,23 +13,49 @@
 use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
 
 /**
- * This class sequentially writes rows from the database to a CSV file + returns max value of inc. fetching column.
+ * Writes rows from the database to a CSV file + returns max value of inc. fetching column.
  */
-class QueryResultCsvWriter
+class DefaultResultWriter implements ResultWriter
 {
     public const MAX_VALUE_STATE_KEY = 'lastFetchedRow';
 
     protected array $state;
 
+    protected ?string $fromEncoding = null;
```

Comment:
No a do `DefaultResultWriter` som pridal moznost prevadzat kodovanie.
Kedze niektore ODBC drivery umoznuju nastavit kodovanie, ine maju pevne dane iba jedno kodovanie, vacsinou `8859-1`. 

By default je to vypnute a nic sa nemeni.